models/Post.js

```diff
@@ -0,0 +1,37 @@
+const mongoose = require('mongoose');
+const Schema = mongoose.Schema;
+
+const PostSchema = new Schema({
+  user: {
+    type: Schema.Types.ObjectId,
+    ref: 'User'
+  },
+  content: {
+    type: String,
+    required: true
+  },
+  title: {
+    type: String
+  },
+  date: {
```

Comment:
I see that you used this field to sort your notes, but I don't see where your `put` method updates the date. Will the notes always be sorted by the order in which they were created? Or do you want to update the `date` field as part of your `PUT` request, which would allow you to sort them based on when the note was last edited?

Maybe it would be useful to have separate `createdAt` and `updatedAt` fields for this schema, if you wanted access to both pieces of information?